### PR TITLE
Updating egotrip bless

### PR DESCRIPTION
**Pull request by Durhamite2118**
including egotrip bless for onslaught

### CODE REVIEW

File: code/code/misc/spell_parser.cc [hunk 1]
```diff
@@ -1673,7 +1673,7 @@ namespace {
     { AFFECT_VASCO_BLESSING, "AFFECT_VASCO_BLESSING" },
     { AFFECT_CORAL_BLESSING, "AFFECT_CORAL_BLESSING" },
     { AFFECT_BUMP_BLESSING, "AFFECT_BUMP_BLESSING" },
-    { AFFECT_MAROR_BLESSING, "AFFECT_MAROR_BLESSING" },
+    { AFFECT_ONSLAUGHT_BLESSING, "AFFECT_ONSLAUGHT_BLESSING" },
```
Comment (cizra) [inline code/code/misc/spell_parser.cc:1676]: Why did you feel a need to kick Maror out?

**Review by cizra — CHANGES_REQUESTED** [review]

File: code/code/misc/spell_parser.cc (hunk 1 above)
Comment (Durhamite2118) [reply code/code/misc/spell_parser.cc:1676]: Was following Warluks suggestion to replace a current blessing for simplicity.  Maror was a random choice.

**Review by Durhamite2118 — COMMENTED** [review]
Removed dead code.

File: code/code/misc/spell_parser.cc (hunk 1 above)
Comment (cizra) [reply code/code/misc/spell_parser.cc:1676]: Eh, why not. I guess it doesn't matter much.

**Review by cizra — APPROVED** [review]

**Pull request merged**